tests/components/image/test_init.py

```diff
@@ -324,4 +325,8 @@ async def test_image_stream(
         await hass.async_block_till_done()
         await hass.async_block_till_done()
 
+    # Wait for the write
+    await asyncio.sleep(0)
+    # Wait for the close
+    await asyncio.sleep(0)
```

Comment:
I don't really like the sleeps here, but its the same thing we do in aiohttp tests so probably ok

The alternative is patching aiohttp which seemed less desirable.

Reply:
I came up with a better way